### PR TITLE
Created "About me" page

### PR DESCRIPTION
**Pull request by iboved**
about

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2,9 +2,39 @@
...
 <body>
-
+    <header>
+        <h1>Volodymyr Gavrylov</h1>
+    </header>
+    <article>
+        <div class="container">
+            <div class="description">
+                <p>Hi!</p>
+                <p>My name is <b>Volodymyr Gavrylov</b>. I am 25 years old. I studied at the <i>GeekHub Advanced PHP</i> course and successfully completed it. This gave me a big boost in web development. Now I am a student of the <i>GeekHub Frontend + JS</i> course. And this is a good opportunity for me to get new knowledge. I want to grow up and become a full stack developer. I hope this course will help me to get the desired skills in frontend development.</p>
```
Comment (spolischook) [inline index.html]: Add line breaks

**Review by spolischook — APPROVED** [review]
LGTM

File: index.html (hunk 1 above)
Comment (iboved) [reply index.html]: Is it a good practice to use line breaks in `<p>` or other tags if the lines are very long?

Comment (iboved) [reply index.html]: Fixed.

**Pull request merged**